test/skip.test.js

```diff
@@ -0,0 +1,12 @@
+const { exec } = require('./exec');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/codingstyle-cn/hlj/pullRequest?prid=7143429)
```suggestion
const { exec } = require("./exec");
```